packages/cli-lib/lang/en.lyaml

```diff
@@ -565,7 +565,24 @@ en:
                 describe: "Name to use for created sandbox"
             success:
               create: "Sandbox \"{{ name }}\" with portalId \"{{ sandboxHubId }}\" created successfully."
+              prePrompt: "
+
+
+              The following step will prompt you to authenticate with sandbox \"{{ name }}\" and add it to the CLI config.
```

Comment:
do they need to take an action to get the the following step?

Reply:
Same as above - addressed in a later commit and this section has been removed